docker-compose-snapshots-mysql-binlog.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  orderhistorytextsearchservice:
+    build: ./order-history-text-search-service/
+    ports:
+      - "8084:8080"
+    depends_on:
+      - mysql
```

Comment:
I don't think this needs MySQL.
Duplicate detection if needed should be implemented within ElasticSearch

Reply:
Fixed